praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/domain/mappers/PraktikumsstellenMapper.java

```diff
@@ -38,4 +42,35 @@ AusbildungsPraktikumsstelle toEntity(final CreateAusbildungsPraktikumsstelleDto
     @Mapping(target = "id", ignore = true)
     @Mapping(target = "assignedNwk", ignore = true)
     AusbildungsPraktikumsstelle toEntity(final CreateAusbildungsPraktikumsstelleWithMeldezeitraumDto createAusbildungsPraktikumsstelleWithMeldezeitraumDto);
+
+    @Mapping(target = "id", source = "id")
+    AusbildungsPraktikumsstelle toEntity(final UUID id,
+            final UpdateAusbildungsPraktikumsstelleWithMeldezeitraumAndAssignedNWKDto ausbildungsPraktikumsstelleDto);
+
+    @Mapping(target = "id", source = "id")
+    StudiumsPraktikumsstelle toEntity(final UUID id, final UpdateStudiumsPraktikumsstelleWithMeldezeitraumAndAssignedNwkDto studiumsPraktikumsstelleDto);
+
+    @Mapping(target = "dienststelle", source = "praktikumsstelleDto.dienststelle")
+    @Mapping(target = "oertlicheAusbilder", source = "praktikumsstelleDto.oertlicheAusbilder")
+    @Mapping(target = "email", source = "praktikumsstelleDto.email")
+    @Mapping(target = "taetigkeiten", source = "praktikumsstelleDto.taetigkeiten")
+
+    @Mapping(target = "id", ignore = true)
+    @Mapping(target = "projektarbeit", ignore = true)
+    @Mapping(target = "ausbildungsjahr", ignore = true)
+    @Mapping(target = "ausbildungsrichtung", ignore = true)
+    AusbildungsPraktikumsstelle updateAusbildungsPraktikumsstelle(@MappingTarget final AusbildungsPraktikumsstelle ausbildungsPraktikumsstelle,
+            final UpdateAusbildungsPraktikumsstelleWithMeldezeitraumAndAssignedNWKDto praktikumsstelleDto);
+
+    @Mapping(target = "dienststelle", source = "praktikumsstelleDto.dienststelle")
+    @Mapping(target = "oertlicheAusbilder", source = "praktikumsstelleDto.oertlicheAusbilder")
+    @Mapping(target = "email", source = "praktikumsstelleDto.email")
+    @Mapping(target = "taetigkeiten", source = "praktikumsstelleDto.taetigkeiten")
+
```

Comment:
I don't think the empty line makes the annotations more readable. The existing grouping is useful enough

```suggestion

```